src/atoms/Space/Space.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, text } from '@storybook/addon-knobs';
+import Text from '../Text';
+import Space from './Space';
+
+const sizeOptions = {
+  xxsmall: 'xxsmall',
+  xsmall: 'xsmall',
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+};
+
+storiesOf('Space', module)
+  .addParameters({
+    component: Space,
+  })
+  .add('default', () => (
+    <Space margin={[1, 1]} padding={[2, 2]}>
+      <Text _weight="bold" _isUnderlined size={select('Size', sizeOptions, 'medium')} as="span">
```

Comment:
Don't use any underscore ```_``` props unless necessary(from design)

Reply:
But it's a part of Text component

Reply:
All ```_``` props are called "private props" which are meant to be used only as an exception from design

Reply:
Private props usage shouldn't be part of stories.